lib/theme_javascript_compiler.rb

```diff
@@ -210,24 +210,34 @@ def append_ember_template(name, hbs_template)
 
   def raw_template_name(name)
     name = name.sub(/\.(raw|hbr)\z/, "")
```

Comment:
what's the point of reassigning name now?

Reply:
Thanks!